glossary.yml

```diff
@@ -299,7 +304,14 @@
     def: >
       A set of functions and procedures provided by one software library or web
       service through which another application can communicate with it. An API is not
-      the code, the database, or the server: it is the access point.
+      the code, the database, or the server: it's the access point.
+  af:
+    term: "Toepassing Programmeerderkoppelvlak"
+    def: >
+      n Versameling funksies en prosedures wat deur 'n sagtewareprogrammateek
```

Comment:
sagtewareprogrammateek na sagtewarebiblioteek

Reply:
Eintlik verkies ek sagtewareprogrammateek. Die woord biblioteek kom vanaf die Grieks,  βιβλιοθήκη (bibliothḗkē, “boek-kamer”), van βιβλίον (biblíon, “book”) en‎ θήκη (thḗkē, “kis”). Ek sou dus sê dat 'n sagtewarebiblioteek verwys na 'n versameling boeke wat handel oor sagteware, terwyl 'n programmateek na 'n versameling programme verwys. Die woord programme, is nie uniek aan sagteware nie en dus maak die byvoeging van die woord sagteware, die woord spesifiek.

Reply:
Klink dan goed vir my!/ Sounds good to me!